test/fixtures/mucked_homes.yml

```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+# This model initially had no columns defined. If you add columns to the
+# model remove the '{}' from the fixture names and add the columns immediately
+# below each fixture, per the syntax in the comments below
+#
+one: {}
```

Comment:
We use fixtures pretty extensively in the app; let's have 2 filled out

Reply:
I don't even know what fixtures are :x These were generated when I created the model I think? I'll take a look at existing fixtures and see if I can figure out what to do with the new ones...